### PR TITLE
Add 3 new endpoints: Flight Most Booked Destinations, Flight Busiest Period & Airline Code Lookup

### PR DESCRIPTION
**Pull request by samheyman**
Add endpoint implementations for the new following APIs:
- Airline Code LookUp
- Flight Busiest Traveling Period
- Flight Most Booked Destinations

### CODE REVIEW

**Comment by samheyman** [comment]
@anthonyroux ready for review!

File: lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            #
+            # @option params [String] :origin IATA code of the origin city -
+            #   e.g. BOS for Boston - required
+            # @option params [String] :query period when consumers
+            # are travelling
+            #   in YYYY-MM format
+            # @return [Amadeus::Response] a parsed response
+            # @raise [Amadeus::Base] an exception if the call failed
+            # @example Find the air traffic from LHR in January 2011
+            #   amadeus.travel.analytics.air_traffic.booked.get(
```
Comment (anthonyroux) [inline lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb]: Wrong copy/past

File: lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    class Travel
+      class Analytics
+        class AirTraffic
+          # A namespaced client for the
+          # +/v1/travel/analytics/air-traffic/booked+ endpoints
+          #
+          # Access via the +Amadeus::Client+ object
+          #
+          #   amadeus = Amadeus::Client.new
+          #   amadeus.travel.analytics.air_traffic.booked
```
Comment (anthonyroux) [inline lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb]: wrong copy/past

File: lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```
Comment (anthonyroux) [inline lib/amadeus/namespaces/travel/analytics/air_traffic/busiest_period.rb]: Please review the comments, you have the ones from most booked

**Pull request merged**